test/requests/error_json_test.rb

```diff
@@ -67,12 +93,12 @@ def without_layouts
     `mv error.html.erb test/fake_app/app/views/layouts/`
   end
 
-  def get(path)
+  def get(path, headers: {})
```

Comment:
I think this is a good change so the signature will be more aligned with , but it would be great if you could send a separate pull request for that.

```ruby
def get(path, params: {}, headers: {})
  ...
end
```

Reply:
That's a good idea. Although I do need it in order to have more flexibility testing with GET since you overwrote it. I'll create a separate PR, and then see if I can rebase this PR off the other one.

Reply:
see #56 (merged onto this PR)